include/caf/leds.rst

```diff
@@ -10,27 +10,59 @@ CAF: LEDs module
 The LEDs module of the :ref:`lib_caf` (CAF) is responsible for controlling LEDs in response to LED effect set by a ``led_event``.
 The source of such events could be any other module in |NCS|.
 
-The module uses its own variant of Zephyr's :ref:`zephyr:pwm_api` driver for setting the LED color, either RGB or monochromatic.
-For this reason, it requires a manual configuration of the PWM ports and LEDs.
+The module uses Zephyr's :ref:`zephyr:led_api` driver for setting the LED color, either RGB or monochromatic.
+The Zephyr's LED driver can use either GPIO or PWM based implementation.
 
 Configuration
 *************
 
 To use the module, you must enable the following Kconfig options:
 
 * :option:`CONFIG_CAF_LEDS` - This option enables the LEDs module.
-* :option:`CONFIG_PWM` - This option enables Zephyr's :ref:`zephyr:pwm_api` driver, which is used for setting the LED color (that is, the brightness of the diodes).
 * :option:`CONFIG_LED` - This option enables the LED driver API.
-* :option:`CONFIG_LED_PWM` - This option enabled the LED PWM driver.
 
-Additionally, you need to complete the steps described in `Configuring PWM ports and LEDs`_.
+Depending on selected LED driver implementation, you must also enable one of the following sets of options:
+
+* :option:`CONFIG_CAF_LEDS_PWM`, :option:`CONFIG_LED_PWM` and :option:`CONFIG_PWM`
+  These options are required for PWM based implementation.
+  Zephyr's :ref:`zephyr:pwm_api` driver is used for setting the LED color (that is, the brightness of the diodes).
+* :option:`CONFIG_CAF_LEDS_GPIO`, :option:`CONFIG_LED_GPIO` and :option:`CONFIG_GPIO`
+  These options are required for GPIO based implementation.
+  Zephyr's :ref:`zephyr:gpio_api` driver is used for setting the LED color (that is, the brightness of the diodes).
```

Comment:
Can we say the brightness here if its only on/off?

Reply:
There is a note with more detailed explanation of this limitation. The CAF module still uses API to set the LED brightness (but it actually can set the brightness either to 100% or 0% if GPIO LED driver is in use).